controller/workitem_blackbox_test.go

```diff
@@ -41,6 +41,11 @@ import (
 	"github.com/stretchr/testify/suite"
```

Comment:
_And why is it ID = "0" which sounds like a fairly magic string as oppose to ID = nil?_

The ID = "0" and not "nil" because if transaction fails because of Internal Error then tx.Error returns Item.ID == "nil". I need to differentiate between the case where transaction fails and the case where record is not found.